CHANGELOG.md

```diff
@@ -45,6 +45,9 @@ to behave like `helm-find-files`, such as multifile selection and opening or del
 * Prevent `projectile-kill-buffers` from trying to kill indirect
 buffers.
 * [#412](https://github.com/bbatsov/projectile/issues/412): Handle multiple possible targets in `projectile-toggle-between-implementation-or-test`.
+* Use `file-truename` when caching filenames to prevent
```

Comment:
0.12 was already released. See the latest version of the changelog and you'll understand what I mean.